src/aggregator/aggregator/follower_flush_mgr.go

```diff
@@ -267,7 +286,21 @@ func (mgr *followerFlushManager) CanLead() bool {
 			if waitTillFlushedTime.Equal(mgr.openedAt) {
 				waitTillFlushedTime = waitTillFlushedTime.Add(-windowSize)
 			}
-			for _, lastFlushedNanos := range fbr.ByNumForwardedTimes {
+
+			for numForwardedTimes, lastFlushedNanos := range fbr.ByNumForwardedTimes {
+				if lastFlushedNanos == 0 {
+					mgr.logger.Info("Encountered a window that was never flushed by leader",
+						zap.Stringer("windowSize", windowSize),
```

Comment:
Nit: let's log `now` as well? Might as well, and I think with that we can fully determine what's going on.